docs/patterns/components/Tooltip/index.js

```diff
@@ -178,7 +178,7 @@ const Demo = () => {
           },
           show: {
             type: 'boolean',
-            description: 'Manually control the show state of the tooltip'
+            description: 'Whether or not the tooltip is shown initially.'
```

Comment:
See: https://github.com/dequelabs/cauldron/issues/546

Reply:
I'm debating on if this should be a breaking change or not - the description implies that it changes the functionality of the prop. If so, I _think_ this PR should have _BREAKING CHANGE_ because we're changing the original intended functionality. I don't think we use it like this anywhere internally, but it's possible someone else was.

Reply:
Right, but if someone is using this component based on the current expected documented functionality, it still is a breaking change because we're changing the documented functionality.

Reply:
So we should either:

* Fix the issue to match what is currently documented
* Or keep the code as is and mark this change as a breaking change

Reply:
I reverted the description and added a note that there is a known issue.